rio_tiler/io/rasterio.py

```diff
@@ -545,6 +546,21 @@ def feature(
             **kwargs,
         )
 
+        if shape is not None:
+            cutline_mask = rasterize(
+                [shape],
+                out_shape=(img.height, img.width),
+                transform=img.transform,
+                all_touched=kwargs.get("all_touched", False),
```

Comment:
🤔 not sure about this. can you explain?

Reply:
This is a gdal option for how to rasterize the geometry. From [gdal docs](https://gdal.org/programs/gdal_rasterize.html)

> Enables the ALL_TOUCHED rasterization option so that all pixels touched by lines or polygons will be updated, not just those on the line render path, or whose center point is within the polygon. Defaults to disabled for normal rendering rules.

See also
https://rasterio.readthedocs.io/en/latest/api/rasterio.features.html#rasterio.features.rasterize

I was insecure about the "api" here, we could also add this as an explicit kwarg to the `feature` function.

Reply:
yeah I think it will make more sense to have a specific option in the `feature` method.